articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,127 @@
+
+**Android**: if you are using a custom scheme, you must pass this scheme to the login method so that the SDK can route to the login page and back again correctly:
+
+```dart
+await auth0.webAuthentication.login(scheme: 'YOUR CUSTOM SCHEME');
+```
+
+Once a user has logged in, they will be redirected back to your application. You will then be able to access the ID and access tokens for this user.
+
+::::checkpoint
+:::checkpoint-default
+Add a button to your app that calls `webAuthentication.login()` and logs the user into your app. Verify that you are redirected to Auth0 for authentication, and then back to your application.
+
+Verify that you can get access to the tokens on the result of calling `login`.
+:::
+
+:::checkpoint-failure
+If your application did not launch successfully:
+* Ensure you set the Allowed Callback URLs are correct
+* Verify you saved your changes after entering your URLs
+* Make sure the domain and client ID values are imported correctly
+* If using Android, ensure that the manifest placeholders have been set up correctly, otherwise the redirect back to your app may not work
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Show user profile information {{{ data-action="code" data-code="main.dart#8:9" }}}
+
+The user profile is automatically retrieved for you when you call `webAuthentication.login()`. The object that is returned from the login step contains a `userProfile` property that has all the user profile properties, which have been populated by decoding the ID token.
+
+::::checkpoint
+:::checkpoint-default
+Log in and inspect the `userProfile` property on the result. Verify that you can access the current user's profile information.
+:::
+:::checkpoint-failure
+If your application did not return user profile information:
+
+- Verify the access token is valid
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Add logout to your app {{{ data-action=code data-code="main.dart#11:12"}}}
+
+To log out, users must be redirected to the Auth0 logout endpoint to clear their login session. This can be achieved using the Auth0 Flutter SDK using `webAuthentication.logout()`. [Read more about logging out of Auth0](https://auth0.com/docs/authenticate/login/logout).
+
+**Android**: if you are using a custom scheme, you must pass this scheme to the logout method so that the SDK can route back to your app correctly:
+
+```
+await auth0.webAuthentication.logout(scheme: 'YOUR CUSTOM SCHEME');
+```
+
+::::checkpoint
+:::checkpoint-default
+Add a button to your app that calls `logout` and logs the user out of your application. When you click it, verify that your Flutter app redirects you to the logout endpoint and back again and that you are no longer logged in to your application.
```

Comment:
In the checkpoint for login it refers to the login function as `webAuthentication.login()`. Should we also refer to `logout` the same way here?